sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDirectorySteps.java

```diff
@@ -127,14 +141,269 @@ public CaseDirectorySteps(
               "Total number of displayed cases doesn't match with the number of cases created via api");
         });
 
+    And(
+        "I apply Person Id filter",
+        () ->
+            webDriverHelpers.fillAndSubmitInWebElement(
+                PERSON_ID_NAME_CONTACT_INFORMATION_LIKE_INPUT,
+                apiState.getLastCreatedPerson().getFirstName()
+                    + " "
+                    + apiState.getLastCreatedPerson().getLastName()));
+    And(
+        "I apply Random Person Id filter",
```

Comment:
same, location in step name and faker cand generate a valid name, so let's hardcode a value which won't exist.